addons/event_sms/models/event_registration.py

```diff
@@ -1,13 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import models
-
-
-class Registration(models.Model):
-    _inherit = 'event.registration'
-
-    def _sms_get_number_fields(self):
```

Comment:
Mmh, shouldn't we override "_phone_get_number_fields" then? Looks like default implementation returns an empty list. Maybe it was already done elsewhere, a bit lost ^^